lib/cli_interface.rb

```diff
@@ -26,20 +24,34 @@ def set_lang(input)
     @text_file = YAML::load(File.open(file))
   end
 
-  def menu_game
-    write("#{get_from_file('menu_type_games')}")
+  def display_menu_type_game(default_checked_type = 1)
+    display_menu(get_from_file('menu_type_games'), default_checked_type)
   end
 
-  def menu_lang
-    display_lang_files
+  def menu_type_game_size
+    get_from_file('menu_type_games').lines.size - 1
   end
 
-  def menu_first_player
-    write("#{get_from_file('menu_first_player')}")
+  def display_menu_lang(default_checked_lang = 1)
+    menu_lang = get_menu_lang
```

Comment:
+1 just call the method itself `menu_lang`. Having `get_*` or `set_*` is not idiomatic Ruby.